cli/__snapshots__/verify_spec.ts.js

```diff
@@ -23,10 +23,6 @@ exports['current version has not been verified 1'] = `
 It looks like this is your first time using Cypress: 1.2.3
 
 
-⠋  Verified Cypress! /cache/Cypress/1.2.3/Cypress.app
```

Comment:
the verify printer is silent within these tests now as it is too non deterministic on printout with the refactor. My guess is there is slight timing differences between running a raw JavaScript file and `ts-node` loading and interpreting a TS file, but could be wrong.